tapioca/tapioca.py

```diff
@@ -97,8 +97,8 @@ def __dir__(self):
     def __str__(self):
         import pprint
         pp = pprint.PrettyPrinter(indent=4)
-        return """<{} object
-{}>""".format(self.__class__.__name__, pp.pformat(self._data))
+        return "<{} object {}>".format(
+            self.__class__.__name__, pp.pformat(self._data))
```

Comment:
This line break is on purpose. There should be a break after "object" and the data to facilitate reading.